### ISSUE TITLE
Bug in subscriber counter

### ISSUE THREAD

**Issue by AutomCoding** [Chocobozzz/PeerTube#1256]
**What happened?**
The counter shows a negative subscriber count at [this page](https://peertube.social/video-channels/51f2a02f-29ae-4b8b-a711-d8567f966710@video.blender.org/videos).
![namnlos](https://user-images.githubusercontent.com/40275136/46904741-6e6ae080-cee9-11e8-967b-d84ba07b35e2.png)

**What do you expect to happen instead?**
This should only be a positive integer.

**Additional information**
Local PeerTube version: PeerTube v1.0.0
Remote PeerTube version: PeerTube v1.0.0-beta.8

**Comment by AutomCoding** [comment]
That channel has now been deleted, but is still visible at [this page](https://peertube.social/accounts/blender@video.blender.org/video-channels) and not on [the corresponding page](https://video.blender.org/accounts/blender/video-channels) at the home instance.

I don't know if the channel was deleted already when I created this issue, so the problem could perhaps be something concerning synchronization of channel deletion as it would explain both issues. If that is the case there is also some uncaught error when unsubscribing from a deleted remote channel, allowing unsubscriptions to register locally but not subscriptions.

The hosting instance ([video.blender.org](https://video.blender.org/)) has now been updated to version 1.0.1 and the remote ([peertube.social](https://peertube.social/)) to 1.1.0.
